packs/feat-effects/effect-hydration.json

```diff
@@ -23,7 +23,7 @@
         "rules": [
             {
                 "key": "FlatModifier",
-                "label": "Dehydrated",
+                "label": "PF2E.SpecificRule.Azarketi.Hydration.Label",
```

Comment:
Maybe the effect should be renamed to "Effect: Dehydrated"? Then we wouldn't need the localized label?

Reply:
Yeah, let's go with that.
Makes more sense.